hugo/content/lessons/testing-cloud-functions-in-firebase/index.md

```diff
@@ -290,4 +280,4 @@ describe('makePayment', () => {
 
 ## The End
 
-There are many different possibilities when it comes to testing Cloud Functions. Ultimately, the right strategy depends on the critical business needs of the app, but live testing with Jest is a reliable approach for many situations. Let me know if you want to see any other testing strategies in the comments.  
\ No newline at end of file
+There are many different possibilities when it comes to testing Cloud Functions. Ultimately, the right strategy depends on the critical business needs of the app, but live testing with Jest is a reliable approach for many situations. Let me know if you want to see any other testing strategies in the comments.  
```

Comment:
I made the whole edit inline in Github and it's marking this as a change despite not touching these lines. The content is identical so it might have just automatically added a newline at the end.